src/vs/code/electron-main/window.ts

```diff
@@ -510,6 +510,13 @@ export class CodeWindow implements ICodeWindow {
 			this._readyState = ReadyState.NAVIGATING;
 		}
 
+		// Add args to the config, but do not preserve them on currentConfig or
+		// pendingLoadConfig so that they are applied only on this load
+		const configuration = objects.assign({}, config);
+		if (temporaryArgs) {
+			configuration['disable-extensions'] = temporaryArgs['disable-extensions'];
```

Comment:
It is a bit weird that we allow `temporaryArgs` to be any `ParsedArgs` but then here only one is supported. Should we not support all? Or if not, why don't we just add an explicit boolean to this method to disable extensions once?